source/PlayerInfo.cpp

```diff
@@ -2554,6 +2576,73 @@ void PlayerInfo::SetTravelDestination(const Planet *planet)
 
 
 
+void PlayerInfo::QueueRelocation(const Planet *destination, bool flagshipOnly)
+{
+	relocation = Relocation(destination, flagshipOnly);
+}
+
+
+
+void PlayerInfo::DoQueuedRelocation()
+{
+	if(!relocation.relocationPlanet || !FlagshipPtr())
+		return;
+	if(!planet)
+	{
+		relocation.relocationPlanet = nullptr;
+		return;
+	}
+
+	Visit(*relocation.relocationPlanet->GetSystem());
+	Visit(*relocation.relocationPlanet);
+
+	flagship->SetSystem(relocation.relocationPlanet->GetSystem());
+	flagship->SetPlanet(relocation.relocationPlanet);
+	if(!relocation.relocateFlagshipOnly)
+		for(const shared_ptr<Ship> &ship : ships)
+			if(!ship->IsParked() && !ship->IsDestroyed() && ship->GetPlanet() == planet)
+			{
+				ship->SetSystem(relocation.relocationPlanet->GetSystem());
+				ship->SetPlanet(relocation.relocationPlanet);
+			}
+	system = relocation.relocationPlanet->GetSystem();
+	planet = relocation.relocationPlanet;
+	relocation.relocationStatus = RelocateStatus::IN_PROGRESS;
+	relocation.oldRelocationPlanet = relocation.relocationPlanet;
```

Comment:
What's the purpose of this variable? The player's planet and the oldRelocationPlanet are now the same value.